adapters/appnexus_test.go

```diff
@@ -276,7 +276,8 @@ func TestAppNexusBasicResponse(t *testing.T) {
 		}
 
 		pbin.AdUnits[i] = pbs.AdUnit{
-			Code: tag.code,
+			Code:       tag.code,
+			MediaTypes: []string{"BANNER"},
```

Comment:
should test that if you only ask for banner you only get a banner, and probably also add a test for banner + video